app/services/notifications/setup/review.service.js

```diff
@@ -17,15 +20,32 @@ const SessionModel = require('../../../models/session.model.js')
 async function go(sessionId) {
   const session = await SessionModel.query().findById(sessionId)
 
-  // eslint-disable-next-line no-unused-vars
   const { returnsPeriod } = session
+  const selectedReturnsPeriod = _extractReturnPeriod(returnsPeriod)
+  const summer = _summer(returnsPeriod)
+
+  const recipients = await RecipientsService.go(selectedReturnsPeriod.dueDate, summer)
+
+  const pageData = ReviewPresenter.go(recipients)
```

Comment:
I think this is the new convention the other team have come up with.

```suggestion
  const formattedData = ReviewPresenter.go(recipients)
```